src/request/sign-transaction/BaseLayout.js

```diff
@@ -94,10 +112,31 @@ class BaseLayout {
             },
         );
 
-        // This event cannot throw a 'CANCEL' error like in other requests,
-        // because for checkout we need to go back to the CheckoutOverview
-        // in the Accounts Manager and not return directly to the caller.
-        this._passphraseBox.on(PassphraseBox.Events.CANCEL, () => window.history.back());
+        /** @type {HTMLElement} */
+        this.$accountDetails = (this.$el.querySelector('#account-details'));
+        const $accounts = this.$el.querySelectorAll('.account');
+        $accounts.forEach($item => $item.addEventListener('click', event => this._openDetails($item, event)));
+        /** @type {HTMLButtonElement} */
+        const $closeDetails = (this.$accountDetails.querySelector('#close-details'));
+        $closeDetails.addEventListener('click', this._closeDetails.bind(this));
+        /** @type {HTMLElement} */
+        const $background = (this.$el.querySelector('#background-overlay'));
+        $background.addEventListener('click', this._closeDetails.bind(this));
+    }
+
+    /**
+     * @param {Element} $el
+     * @param {Event} event
+     */
+    _openDetails($el, event) {
+        event.preventDefault();
+        /** @type {HTMLElement} */
+        (this.$accountDetails.querySelector('#details')).innerHTML = $el.innerHTML;
+        this.$el.classList.add('open');
```

Comment:
Needs a better class name if added to the main element (`this.$el`). Like `opened-account-details` or something.